pkg/engine/plugins.go

```diff
@@ -252,6 +252,7 @@ func gatherPackagesFromProgram(plugctx *plugin.Context, runtime string, info plu
 		return nil, fmt.Errorf("failed to load language plugin %s: %w", runtime, err)
 	}
 
+	defer lang.Close()
```

Comment:
It looks like this fails when gathering packages of non-inline programs because there would already be a language host running and this closes it prematurely. Removing this for now